bootScripts/NodeThreadWorkerBootScript/index.js

```diff
@@ -157,6 +159,23 @@ function boot() {
                 }
             }
 
+            if (cacheContainerPath) {
+                try {
+                    dsuCodeFileCacheHandler = new DSUCodeFileCacheHandler(dsu, cacheContainerPath);
+
+                    // construct the cache in parallel since it takes a bit of time
+                    setTimeout(async () => {
```

Comment:
Currently the process of constructing the cache is pretty time intensive (e.g. for leaflet wallet, the /code has ~30 MB with ~3000 files, out of which only a small part is used - less than 100).
On windows with Chrome, the boot process takes around 15 seconds. This means that the leaflet wallet page will take ~17 seconds to appear for the first time, while typically it takes ~3 seconds to load. This is a huge difference for the first load time.

On windows with Android emulator, the boot process takes also around 15 seconds. But also, an ordinary leaflet wallet app will be opened in ~19 seconds. So this cache constructing step is not obvious for Android emulator.

Because of these cases, the cache construction step is done in parallel with serving cloudWallet request. When the cache is constructed, the files will be served from there, otherwise it's served from directly the DSU.
If the first time is not that important, then the *setTimeout* can be removed here, in order to block any cloudWallet requests until the cache is constructed.